lib/rpc-client.js

```diff
@@ -0,0 +1,72 @@
+import {
+  FilecoinRpcError,
+  INVALID_RPC_RESPONSE,
+  NO_RESULT_IN_RESPONSE,
+  RPC_ERROR,
+} from './errors.js'
+
+/**
+ * @param {string} method
+ * @param {unknown[]} params
+ * @param {object} [options]
+ * @param {string} [options.rpcUrl] URL of the RPC server
```

Comment:
```suggestion
 * @param {string} options.rpcUrl URL of the RPC server
```

This parameter isn't optional